languages/en-ZA.js

```diff
@@ -0,0 +1,38 @@
+/*!
+ * numeral.js language configuration
+ * language : English for South Africa
+ * author : Stewart Scott : https://github.com/stewart42
+ */
+(function() {
+    var language = {
+        delimiters: {
+            thousands: ',',
+            decimal: '.'
```

Comment:
This isn't technically correct. The official en-ZA formatting is: grouping (thousands) is space and decimal is comma. Reference: http://www.localeplanet.com/icu/en-ZA/